helpers/postgresql_dump.sh

```diff
@@ -1,12 +1,34 @@
 #!/bin/bash
+# Script to create backup files for rhel6 to 7 migration.
+#
 # Notes:
-# 1. Use this script to create postgresql dumps - foreman.dump and candlepin.dump from the Satellite server.
-# 2. This scripts stops the katello-service in Satellite.  Please plan accordingly for the outage.
+# 1. This script runs katello-backup and also creates foreman.dump and candlepin.dump files from your Satellite6 server.
+# 2. Run this script as root user in Satellite6 server.
+# 3. This script stops the katello-service in Satellite.  Please plan accordingly for the outage.
+
+set -x
+
+# Create backup folder
 export backup_dir="/backup"
 mkdir $backup_dir
+
+# Run katello-backup
 katello-backup $backup_dir
+echo "pgsql_data.tar.gz is not required for rhel6 to 7 migration..."
+rm -f $backup_dir/katello-backup*/pgsql_data.tar.gz
```

Comment:
@johnpmitsch this works but relying on the word position of 9 in awk command may not be a reliable way to run in production.  I would rather leave it with a text warning.